### PR TITLE
Plane master handling + new chemical Rotatium

### PR DESCRIPTION
**Pull request by MrPerson**
:cl:
add: New toxin - Rotatium. It doesn't quite do what it does on other servers. Instead it rocks your game view back and forth while slowly doing toxin damage. The rocking gets more intense with time.
add: To make it, mix equal parts mindbreaker, neurotoxin, and teslium.
/:cl:

A whole system to create, store, and access screen master objects from the hud. In case anybody actually wants to do anything relevant with them. Which will probably be me but whatever.

Moves things out to some planes that make sense.
click catcher -99
game 0
lighting 15
fullscreen 18
HUD 19
above HUD 20

Rotatium is more of a demonstration than something I really really love, but it is kinda neat so here you go. 

~~No pictures right now, I'll get some later. Sorry.~~
Excuse the low framerates. It looks much nicer ingame.

testingrotatium

testingrotatium2

### CODE REVIEW

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -346,23 +346,19 @@
...
-/obj/screen/click_catcher/proc/MakeGreed()
-	. = list()
-	for(var/i = 0, i<15, i++)
-		for(var/j = 0, j<15, j++)
-			var/obj/screen/click_catcher/CC = new()
-			CC.screen_loc = "NORTH-[i],EAST-[j]"
-			. += CC
+/obj/screen/click_catcher/New()
+	..()
+	transform = matrix(200, 0, 0, 0, 200, 0)
```
Comment (optimumtact) [inline code/_onclick/click.dm]: what does this transform do?

File: code/datums/action.dm [hunk 2]
```diff
@@ -138,10 +138,13 @@
 		..(current_button)
 	else if(target)
 		var/obj/item/I = target
-		var/old = I.layer
+		var/old_layer = I.layer
+		var/old_plane = I.plane
 		I.layer = FLOAT_LAYER //AAAH
+		I.plane = FLOAT_PLANE //^ what that guy said
```
Comment (optimumtact) [inline code/datums/action.dm]: I'm curious as to what parent comment is referring too?

File: code/game/machinery/overview.dm [hunk 3]
```diff
@@ -179,6 +179,7 @@
 		qdel(J)
 		H.icon = HI
 		H.layer = ABOVE_HUD_LAYER
+		H.plane = ABOVE_HYD_PLANE
```
Comment (optimumtact) [inline code/game/machinery/overview.dm]: HYD

File: code/modules/client/client_procs.dm [hunk 4]
```diff
@@ -218,7 +218,6 @@ var/next_external_rsc = 0
 
 	if(!void)
 		void = new()
-		void = void.MakeGreed()
```
Comment (optimumtact) [inline code/modules/client/client_procs.dm]: removed?

**Review by optimumtact — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
Also for anyone else who might be interested in learnign more about planes
http://www.byond.com/forum/?post=2134589

File: code/_onclick/click.dm (hunk 1 above)
Comment (MrPerson) [inline code/_onclick/click.dm]: Make it 200 times as big as a single tile. It was bothering me during debugging when I was trying to find the plane master objects in client.screen and it was like 50 click catcher objects.

I actually forgot I had done this. If you want it in another PR, that's fair.

File: code/game/machinery/overview.dm (hunk 3 above)
Comment (MrPerson) [inline code/game/machinery/overview.dm]: It compiled locally wtf

**Review by optimumtact — APPROVED** [review]

**Comment by MrPerson** [comment]
Actually I'm not going to be able to record a video of this. Baaaaaah.

Ignore travis btw

**Comment by PKPenguin321** [comment]
why not

**Comment by MrPerson** [comment]
I get a comically low framerate trying to capture while spessmans is running

**Comment by PKPenguin321** [comment]
what program are you using to record?

**Comment by MrPerson** [comment]
Tobba claims a similar set of code on goonstation caused various errors for everyone using a nvidia card on 510. The errors went away on the 511 beta, apparently. Once it goes stable, I have no qualms with forcing everyone to upgrade. I'm not ok with forcing everyone onto a beta though.

Def worth a test merge to see if the same problems crop up here.

**Comment by MrPerson** [comment]
Attached some video to OP

**Comment by AnturK** [comment]
Was this testmerged yet?

**Comment by MrPerson** [comment]
Not to my knowledge

**Pull request merged**